plugins/hydra_joblib_launcher/tests/test_joblib_launcher.py

```diff
@@ -67,3 +70,34 @@ def test_example_app(hydra_sweep_runner: TSweepRunner) -> None:
         assert sweep.returns is not None and len(sweep.returns[0]) == 4
         for ret in sweep.returns[0]:
             assert tuple(ret.overrides) in overrides
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "overrides, expected",
+    [
+        ("hydra.launcher.batch_size=1", ("hydra.launcher.batch_size", "1")),
```

Comment:
maybe I am overlooking something but shouldnt the value be int 1 and not "1" ?

Reply:
it is still a string, since we are not altering the Hydra config. the casting happens right before we pass the params to joblib.

The test's main goal is to make sure the example app runs with no issue (without the changes the example app will not run)

Reply:
got it :)